README.md

```diff
@@ -28,36 +28,31 @@ This sample requires the following:  
   * A web server to host the HTML file. This server must support SSL requests.
   * A Microsoft [work or school account](https://dev.office.com/devprogram). This sample relies on a request to get the top 4 contacts for the current user. This request will work only with a work or school account.
   
-## Register the application
-
-1. Sign into the [Application Registration Portal](https://apps.dev.microsoft.com/) using either your personal or work or school account.
-
-2. Choose **Add an app**.
-
-3. Enter a name for the app, and choose **Create application**. 
-    
-   The registration page displays, listing the properties of your app.
-
-4. Copy the Application Id. This is the unique identifier for your app. 
-
-5. Under **Platforms**, choose **Add Platform**.
-
-6. Choose **Web**.
-
-7. Make sure the **Allow Implicit Flow** check box is selected, and enter *https://your host/PeoplePicker.html* as the Redirect URI. 
-
-   The **Allow Implicit Flow** option enables the hybrid flow. During authentication, this enables the app to receive both sign-in info (the id_token) and artifacts (in this case, an authorization code) that the app can use to obtain an access token.
-
-8. Choose **Save**.
+1. Navigate to the [Azure portal - App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) page. 
```

Comment:
Update the entire readme to reflect the changes. This sample no longer uses OfficeJS [helper](https://github.com/microsoftgraph/javascript-officeuifabric-peoplepicker-sample/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R17).